packages/angular-cli/blueprints/universal/files/__path__/server.ts

```diff
@@ -54,7 +57,7 @@ function ngApp(req: any, res: any) {
     req,
     res,
     ngModule: AppModule,
-    preboot: false,
+    preboot: true,
```

Comment:
can you disable preboot? I would like to stick as much as possible to the angular-universal starter.